exoscale/datasource_exoscale_compute_template_test.go

```diff
@@ -11,14 +11,15 @@ import (
 )
 
 var (
-	testAccDataSourceComputeTemplateID       = testInstanceTemplateID
 	testAccDataSourceComputeTemplateName     = testInstanceTemplateName
 	testAccDataSourceComputeTemplateUsername = testInstanceTemplateUsername
 	testAccDataSourceComputeTemplateFilter   = testInstanceTemplateFilter
 	testAccDataSourceTemplateZone            = testZoneName
 )
 
 func TestAccDataSourceComputeTemplate(t *testing.T) {
+	templateID := "4a850c9d-93f4-4b39-b4d7-2cbf3a1f1227"
```

Comment:
Any reason we didn't update original constant `testInstanceTemplateID`?
https://github.com/exoscale/terraform-provider-exoscale/blob/master/exoscale/provider_test.go#L35

Reply:
Actually you did update it as well. It would be best to keep that hardcoded ID in one place, easier to maintain.

Reply:
I'm working on removing it completely (another PR incoming)